easybuild/easyconfigs/n/netCDF/netCDF-4.8.0-gompi-2021a.eb

```diff
@@ -0,0 +1,37 @@
+name = 'netCDF'
+version = '4.8.0'
+
+homepage = 'https://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented
+ scientific data."""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/Unidata/netcdf-c/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['aff58f02b1c3e91dc68f989746f652fe51ff39e6270764e484920cb8db5ad092']
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('CMake', '3.20.1'),
+    ('Doxygen', '1.9.1'),
+]
+
+dependencies = [
+    ('HDF5', '1.10.7'),
+    ('cURL', '7.76.0'),
+    ('Szip', '2.1.1'),
+]
+
+# HDF5 version detection is missed in netCDF 4.8.0 when HDF5_C_LIBRARY, HDF5_INCLUDE_DIR, and HDF5_HL_LIBRARY are set
+_hdf5_version_fix = '-DHDF5_VERSION=1.10.7'
```

Comment:
@branfosj Please use `$EBVERSIONHDF5` here rather than hardcoding?

```python
_hdf5_version_fix = '-DHDF5_VERSION=$EBVERSIONHDF5'
```

I also prefer using `local_hdf5_version_fix` (easier to grep for when needed) rather than using `_` as a prefix for local variables.

Reply:
done in fa86170